encoding.bs

```diff
@@ -999,17 +1000,16 @@ fallback encoding <var>encoding</var>, run these steps:
   than anything else. In a context where HTTP is used this is in violation of the semantics of the
   `<code>Content-Type</code>` header.
 
- <li><p>Let <var>output</var> be the <a for=/>I/O queue</a> of scalar values
- « <a>end-of-queue</a> ».
+ <li><p><a>Run</a> <var>encoding</var>'s <a for=/>decoder</a> with <var>ioQueue</var> and
+ <var>output</var>.
 
- <li><p><a>Run</a> <var>encoding</var>'s
- <a for=/>decoder</a> with <var>ioQueue</var> and <var>output</var>.
+ <li><p><a for="I/O queue">Push</a> <a>end-of-queue</a> into <var>output</var>.
```

Comment:
Or should the "run" algorithm push end-of-queue?

Reply:
As I wrote on IRC, how would you feel about making "process" do it?

Reply:
Works for me